### PR TITLE
fix(dropdown): Fixes miss-aligned dropdown item 

### PR DESCRIPTION
**Pull request by gnas01**
Fixes miss-aligned dropdown item caused by tooltip component

### CODE REVIEW

File: src/components/Dropdown/Dropdown.tsx [hunk 1]
```diff
@@ -269,15 +269,17 @@ const Dropdown: FC<DropdownProps> = ({
...
-            content
-          )}
+          {content}
         </li>
       );
+
+      return item.tooltipContent ? (
+        <Tooltip content={item.tooltipContent} key={`${index}-${item.value}`}>
+          {liElement}
+        </Tooltip>
```
[the remark this answers is not in the document]
Comment (chrisgrigo) [reply src/components/Dropdown/Dropdown.tsx:279]: I tried. It works with disabled as well, this way we don't need to conditionally render item.tooltipContent ? 

However this means that every list item will be surrounded with < div role="tooltip" > (even if it is disabled, i.e when we dont need tooltip). Are we ok with that? If yes, im pushing the change!

[the remark this answers is not in the document]
Comment (SteliosKon) [reply src/components/Dropdown/Dropdown.tsx:279]: Yes, you don't need to render it conditionally. 

I don't see the `< div role="tooltip" >`, is this something that `<Tooltip />` adds internally? I don't think its going to be an issue anyway

[the remark this answers is not in the document]
Comment (chrisgrigo) [reply src/components/Dropdown/Dropdown.tsx]: I pushed the change https://github.com/epignosis/gnosis/pull/418/commits/84e1bc43122ecc2a393a912f6c0dbfeeea15e3ea tests are failing as it is not expecting the < div role="tooltip >, yeah apparently it is added. 

You reckon we should revert to previous version or add the div to tests? also we have to make sure that every dropdown item in etp works as expected with this version right?

[the remark this answers is not in the document]
Comment (chrisgrigo) [reply src/components/Dropdown/Dropdown.tsx]: snapshot update https://github.com/epignosis/gnosis/pull/418/commits/22317a2e851ebf2c31e73a0b3ad47d18d70c3367

**Comment by chrisgrigo** [comment]
Re-opened in https://github.com/epignosis/gnosis/pull/425

**Pull request closed without merging**